### PR TITLE
[Release] Various fixes to release script

### PR DESCRIPTION
**Pull request by Bobgy**
/assign @rmgogogo 
Part of https://github.com/kubeflow/pipelines/issues/3729
Fixes https://github.com/kubeflow/pipelines/issues/3919

### CODE REVIEW

File: sdk/python/requirements.in [hunk 1]
```diff
@@ -10,7 +10,7 @@ jsonschema>=3.0.1
 kubernetes>=8.0.0, <12.0.0
 
 # kfp.Client
-kfp-server-api>=0.2.5, <0.4.0  #Update the upper version whenever a new version of the kfp-server-api package is released. Update the lower version when there is a breaking change in kfp-server-api, or kfp sdk depends on new api changes in kfp-server-api.
+kfp-server-api>=0.2.5, <2.0.0
```
Comment (rmgogogo) [inline sdk/python/requirements.in:13]: +1

**Comment by Bobgy** [comment]
Thanks @rmgogogo !
/hold
wait for travis tests

**Comment by Bobgy** [comment]
These require OWNERS approval, so I'll merge manually first to get fast for release.

**Pull request merged**